tests/TestCase/View/Form/FormContextTest.php

```diff
@@ -206,46 +208,48 @@ public function testAttributes()
      */
     public function testError()
     {
-        $nestedValidator = new Validator();
-        $nestedValidator
-            ->add('password', 'length', ['rule' => ['minLength', 8]])
-            ->add('confirm', 'length', ['rule' => ['minLength', 8]]);
-        $form = new Form();
-        $form->validator()
-            ->add('email', 'format', ['rule' => 'email'])
-            ->add('name', 'length', ['rule' => ['minLength', 10]])
-            ->addNested('pass', $nestedValidator);
-        $form->validate([
-            'email' => 'derp',
-            'name' => 'derp',
-            'pass' => [
-                'password' => 'short',
-                'confirm' => 'long enough',
-            ],
-        ]);
+        $this->deprecated(function () {
+            $nestedValidator = new Validator();
+            $nestedValidator
+                ->add('password', 'length', ['rule' => ['minLength', 8]])
+                ->add('confirm', 'length', ['rule' => ['minLength', 8]]);
+            $form = new Form();
+            $form->validator()
+                ->add('email', 'format', ['rule' => 'email'])
+                ->add('name', 'length', ['rule' => ['minLength', 10]])
+                ->addNested('pass', $nestedValidator);
+            $form->validate([
+                'email' => 'derp',
+                'name' => 'derp',
+                'pass' => [
+                    'password' => 'short',
+                    'confirm' => 'long enough',
+                ],
```

Comment:
This test should be updated to use the non-deprecated features. I'm planning to delete all the tests using `deprecated()` when we move to 4.x development.